### PR TITLE
{io,app-interact}: Update answers for Q2, Q7

### PR DESCRIPTION
**Pull request by Cynerdus**
Signed-off-by: Cleopatra Popescu <cleopatra.popescu02@gmail.com>

### CODE REVIEW

File: app-interact/app-interact-q&a.md [hunk 1]
```diff
@@ -37,6 +37,7 @@
 ### 7. Care este limitarea folosirii pipe-urilor anonime?
 
    Pot întreține comunicarea doar între procese înrudite.
+   De asemenea, sunt dependente de procesele asociate. Un pipe anonim este sters dupa terminarea procesului.
```
Comment (mariasfiraiala) [inline app-interact/app-interact-q&a.md:40]:
```suggestion
   De asemenea, sunt dependente de procesele asociate.
   Un pipe anonim este șters după terminarea procesului, spre deosebire de FIFO-uri care au un back-up în sistemul de fișiere virtual.
```

File: io/io-q&a.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 ### 2. Ce reprezintă un descriptor de fișier?
 
    Descriptorul de fișier este un indice în tabela de file descriptors.
-   Ca intrare pentru un descriptor de fișiere este reținut un pointer către un open file structure.
+   Ca intrare pentru un descriptor de fișiere este reținut un pointer către un open file structure sau alta structura speciala, precum cea de socket, pipe sau terminal.
```
Comment (mariasfiraiala) [inline io/io-q&a.md:10]:
```suggestion
   Ca intrare pentru un descriptor de fișiere este reținut un pointer către un open channel structure.
```

**Review by mariasfiraiala — CHANGES_REQUESTED** [review]
Thanks @Cynerdus! Please apply the changes suggested below and keep only one commit for all of them. For the name, I suggest something along the lines: "{app-interact,io}: Add more details to questions".
Looks really clean, otherwise, we'll merge it quickly.

**Comment by mariasfiraiala** [comment]
@Cynerdus I see that you've integrated my suggestions, however now there are 4 commits to be merged (not elegant at all). Please squash them such that there is only one left with all the changes in it. If you're unsure of how to do this, let me know, I'll rebase your PR myself.

**Comment by Cynerdus** [comment]
@mariasfiraiala Oh yes, thank you for letting me know. It appears I did not read your comment properly, my apologies. Later tonight when I have my unit available, I will squash the commits.

**Comment by mariasfiraiala** [comment]
> @mariasfiraiala Oh yes, thank you for letting me know. It appears I did not read your comment properly, my apologies. Later tonight when I have my unit available, I will squash the commits.

Thank you :heart:

**Comment by Cynerdus** [comment]
@mariasfiraiala Done.

**Review by mariasfiraiala — APPROVED** [review]
Looks alright. Thank you @Cynerdus!

**Pull request merged**